src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -26,9 +20,21 @@ public Window()
 			CoreWindow = new CoreWindow();
 		}
 
-		private void InternalActivate()
+		public int SystemUiVisibility { get; internal set; }
```

Comment:
should be internal since it's not part of UWP contract

Reply:
Actually it's set to 0 by default. That's why I'm combining this SystemUiVisibility  to the decorView one. If one of them hide the navigation, so we will not measure navigation bar. OnSystemUiVisibilityChange is not called on page initialization.

Reply:
I suppose it should be initialized in OnCreate?